### ISSUE TITLE
Wrong Linux Installation Path

### ISSUE THREAD

**Issue by ilyakam** [trakt/Plex-Trakt-Scrobbler#243]
I've spent over an hour trying to figure out why the plugin wasn't showing up in my list of Channels.

The README states it should be extracted and copied to `/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins`
However, the correct path is actually `~/Library/Application Support/Plex Media Server/Plug-ins`

I've verified this on a clean installation of a headless server running Ubuntu 14.04

I'll be happy to submit a PR if you'd like me to.

**Comment by drakulavich** [comment]
I thinks, it's a problem with permissions. Try to set 755 for a plugin folder:
```sudo chmod -R 755 Trakttv.bundle```

**Comment by ilyakam** [comment]
Due to a failed harddrive, I ended up reinstalling everything merely a day later.
Following the link in the `README` had worked this time around.
So first of all, thank you for the advice, @drakulavich!

I've pinpointed the problem:
When I was setting it up the first time around, I was following [this guide](http://www.htpcguides.com/install-plex-media-server-ubuntu-14-04/).
One of the steps requires changing the `PLEX_MEDIA_SERVER_USER` to something other than `plex`
Plex warns that if you do that, you might have to move the Application Support directory.

I still had to `sudo chown plex:plex Trakttv.bundle/ -R`, but that was expected.
I'm going to close this issue, since it was a user error. Thanks for being patient with me.
